chainerx_cc/chainerx/routines/manipulation.cc

```diff
@@ -656,6 +656,11 @@ Array Swapaxes(const Array& a, int8_t axis1, int8_t axis2) {
     return out;
 }
 
+Array Ravel(const Array& a) {
+    Array out = a.Reshape({a.GetTotalSize()});
```

Comment:
Why not write `return a.Reshape({...})` simply?

Reply:
Fixed.